CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Want to hack on the Moby Project? Awesome! We have a contributor's guide that ex
 [setting up a development environment and the contribution
 process](docs/contributing/). 
 
-[![Contributors guide](docs/static_files/contributors.png)](https://docs.docker.com/opensource/project/who-written-for/)
+[![Contributors guide](docs/static_files/contributors.png)](https://docs.docker.com/contribute/overview/)
```

Comment:
This is likely still the wrong link; this guide is for the docs only. We should probably link to `who-written-for.md` or something else instead (or kill the image).

Reply:
OK, sure, we can do that and just update the image